src/Common/Commands.Common/PowerBIConfiguration.cs

```diff
@@ -0,0 +1,56 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using Microsoft.PowerBI.Common.Abstractions.Interfaces;
+
+namespace Microsoft.PowerBI.Commands.Common
+{
+    [DataContract]
+    public class PowerBIConfiguration
```

Comment:
It gets pulled apart so it could be private but I didn't see any harm keeping it public. I get the confusion between settings and configuration, I would like to revisit this but for now I'm blanking better naming without making the change more larger (disruptive)